agentstack/cli/cli.py

```diff
@@ -59,18 +47,27 @@ def configure_default_model():
     agentstack_config = ConfigFile()
     if agentstack_config.default_model:
         log.debug("Using default model from project config.")
-        return  # Default model already set
+        return
 
     log.info("Project does not have a default model configured.")
-    other_msg = "Other (enter a model name)"
```

Comment:
an option here would be to have other list the full list of all models available instead of asking for a string